addons/auth_signup/models/res_users.py

```diff
@@ -225,7 +225,8 @@ def _action_reset_password(self):
                         user.id, force_send=True,
                         raise_exception=True, email_values=email_values)
                 else:
-                    body = self.env['mail.render.mixin']._render_template(
+                    user_lang = user.lang or 'en_US'
```

Comment:
a second fallback could be good, eg. if you are on a website with french / spanish and the user has received the language spanish when visiting the website (from cookies or browser detection) it makes sense to send it in spanish

eg.

```suggestion
                    user_lang = user.lang or self.env.lang or 'en_US'
```

Reply:
yes totally make sense.